bugbug/model.py

```diff
@@ -29,6 +29,14 @@ def __init__(self, lemmatization=False):
 
         self.calculate_importance = True
 
+    @property
+    def le(self):
```

Comment:
I hate this, but I guess it's a necessary evil for now until we make the label encoding part better...

Reply:
We can always move it to the model definition and define it per model.

Reply:
I mean I don't like the fact that we have to access the internal properties :(
I wish the classifiers didn't encode the labels and the responsibility of encoding were of the users of the classifiers.